Resources/Prototypes/Datasets/ion_storm.yml

```diff
@@ -652,7 +669,6 @@
   - OXYGEN TANKS
   - PACKETS
   - PAIS
-  - PANTS
```

Comment:
what's sexist about pants

Reply:
HAVING PANTS IS HARMFUL TO CREW

/!\ erp detected /!\

Reply:
did you gamers not port jumpskirts yet?

Reply:
after thinking about it, this law actually does nothing because the crew doesn't wear pants they wear jumpsuits

Reply:
Could lead to weird outcomes